SourceryRuntime/Sources/Extensions.swift

```diff
@@ -112,8 +112,13 @@ public extension String {
 
     /// :nodoc:
     /// Returns components separated with a comma respecting nested types
-    func commaSeparated() -> [String] {
-        return components(separatedBy: ",", excludingDelimiterBetween: ("<[({", "})]>"))
+    func commaSeparated(ignoringComments: Bool = false) -> [String] {
```

Comment:
rather than introducing a boolean parameter that is only applicable when dealing with enums it might be better to use `componets(separatedBy: excludingDelimiterBetween:)` directly in the `parseEnumAssociatedValues` method.

Reply:
@ilyapuchka works for me, I'll update